test/metabase/models/user_test.clj

```diff
@@ -538,29 +538,32 @@
 (deftest last-used-native-database-id-can-be-read-and-set
   (testing "last-used-native-database-id can be read and set"
     (mt/with-test-user :rasta
-      (let [old-db-id (user/last-used-native-database-id)
-            new-db-id 42]
-        (try
-          (is (not= new-db-id old-db-id))
-          (user/last-used-native-database-id! new-db-id)
-          (is (= new-db-id (user/last-used-native-database-id)))
-          (finally
-            (user/last-used-native-database-id! old-db-id))))))
+      (let [initial-value  (user/last-used-native-database-id)
+            existing-db-id (:id (t2/select-one Database))
+            wrong-db-id    999]
```

Comment:
This is a bit brittle. Can we set a negative value? (Normal DB IDs are positive.)